securing_apps/topics/oidc/fapi-support.adoc

```diff
@@ -13,6 +17,23 @@ To make sure that your clients are FAPI compliant, you can configure Client Poli
 and link them to the global client profiles for FAPI support, which are automatically available in each realm. You can use either `fapi-1-baseline` or `fapi-1-advanced` profile based on which FAPI
 profile you need your clients to conform with.
 
+In case you want to use link:{adminguide_link}#_oidc_clients[Pushed Authorization Request (PAR)], it is recommended that your client use
+both `fapi-1-baseline` profile and `fapi-1-advanced` for PAR requests. Specifically, the `fapi-1-baseline` profile contains `pkce-enforcer` executor, which makes sure
+that client use PKCE with secured S256 algorithm. This is not required for FAPI Advanced clients unless they use PAR requests.
+
+In case you want to use <<_backchannel_authentication_endpoint,CIBA>> in a FAPI compliant way, make sure that your clients use both `fapi-1-advanced` and `fapi-ciba` client profiles.
+There is a need to use `fapi-1-advanced` profile, or other client profile containing the requested executors, as `fapi-ciba` profile contains just CIBA-specific executors.
```

Comment:
Add "the" before `fapi-1-advanced`
change "client profile" to "client profiles"
Add "the" before  `fapi-ciba`